src/quadrature/quadrature.hh

```diff
@@ -224,7 +224,7 @@ Quadrature<dim>::Quadrature(const Quadrature<dim-1> &subq, unsigned int sid, uns
         
         //permute
         for (unsigned int i=0; i<RefElement<dim>::n_nodes_per_side; i++) {
```

Comment:
Why there is the upper bound 'n_nodes_per_side' while the 'p' and 'pp' vectors have size 'dim'? Possibly improve readability by introducing a common constant within this method.

Reply:
For simplex it holds:

RefElement<dim>::n_nodes_per_side = dim

While side_permutations have dimension n_permutations x n_nodes_per_side, points of course have dimension dim. But I do not see which of these 2 labels for the same value should be preferred.

Reply:
Added assert and explanation in the beginning of the constructor.